dbt/models/qc/qc.vw_ic_reference_all_non_res_land_details.sql

```diff
@@ -0,0 +1,55 @@
+SELECT
+    pardat.jur,
+    pardat.taxyr,
+    pardat.parid,
+    {{ insert_hyphens('pardat.class', 1) }} AS pardat_class,
+    {{ insert_hyphens('pardat.tieback', 2, 4, 7, 10) }} AS tieback,
+    legdat.user1,
+    pardat.zoning,
+    {{ insert_hyphens('pardat.nbhd', 2) }} AS nbhd,
+    land.lline,
+    land.acres,
+    land.orate,
+    land.sf,
+    land.brate,
+    land.price,
+    land.ovrprice,
+    {{ insert_hyphens('land.class', 1) }} AS land_class,
+    land.influ,
+    land.code,
+    land.note1,
+    land.allocpct,
+    land.influ2,
+    CONCAT(
+        COALESCE(land.infl1, ''),
+        CASE WHEN land.infl1 IS NOT NULL THEN ':' ELSE '' END,
+        COALESCE(infl1_reascd.description, '')
+    ) AS infl1,
```

Comment:
Might be useful to make a macro out of this logic at some point, but I skipped it for now since we use this logic much less frequently than the `insert_hyphens` logic.